bosh/bosh_ssh.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Can we add a `set -e` to fail if something is missing?